conjur/api/api.py

```diff
@@ -13,8 +13,8 @@
 from datetime import datetime, timedelta
 
 # Internals
-from conjur.endpoints import ConjurEndpoint
-from conjur.http_wrapper import HttpVerb, invoke_endpoint
+from conjur.api.endpoints import ConjurEndpoint
+from conjur.wrappers.http_wrapper import HttpVerb, invoke_endpoint
```

Comment:
wrappers should be wrapper to conform with rest of directory structure

Reply:
Changed to wrapper

Reply:
Done